include/glow/Importer/ONNXIFIModelLoader.h

```diff
@@ -19,16 +19,16 @@
 
 #include "onnx/onnxifi.h"
 
-#include "glow/Importer/ONNX.h"
+#include "glow/Importer/ONNXModelLoader.h"
 
 #include "llvm/ADT/StringMap.h"
 
 namespace glow {
 namespace onnxifi {
 
-class ModelLoader : public ONNXModelLoader {
+class ONNXIFIModelLoader : public ONNXModelLoader {
```

Comment:
it's already in ```onnxifi``` namespace. I think it makes sense to keep a short name for the class.